source/main.cpp

```diff
@@ -214,7 +214,11 @@ int main(int argc, char *argv[])
 		if(!isFullscreen)
 			SDL_GetWindowSize(window, &windowWidth, &windowHeight);
 		
-		
+		// menuPanels is used for the panels related to pilot creation, preferences,
+		// game loading and game saving.
+		// gamePanels is used for the main-panel (flying your spaceship), the panet
```

Comment:
this could be split in two comments, one for each declaration.

Reply:
`panet`